I think the case of combining with `--update-all` wasn't overseen. Here's the traceback I have gotten;
```
==> Finished making: heroic-games-launcher-bin 2.4.0-2 (Mon 15 Aug 2022 03:33:06 PM +03)
Traceback (most recent call last):
  File "/home/beru/local-aur/./local-aur", line 254, in <module>
    build_packages(args.packages, args.update_all,
  File "/home/beru/local-aur/./local-aur", line 118, in build_packages
    new_dest_files = build_package(package)
  File "/home/beru/local-aur/./local-aur", line 71, in build_package
    os.rename(quote(file), quote(dest))
OSError: [Errno 18] Invalid cross-device link: '/tmp/heroic-games-launcher-bin.local-aur.8qm92k6e/heroic-games-launcher-bin-2.4.0-2-x86_64.pkg.tar.zst' -> '/var/lib/local-aur/repo/heroic-games-launcher-bin-2.4.0-2-x86_64.pkg.tar.zst'
sudo ./local-aur build --update-all --install-directly  34.19s user 12.32s system 35% cpu 2:10.78 total
```

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,6 +1,6 @@
 # Maintainer: rpkak <rpkak@users.noreply.github.com>
 pkgname='local-aur'
-pkgver=0.5.0.r1.g446dbaa
+pkgver=0.5.0.r2.g4ca72c0
 pkgrel=1
 epoch=
 pkgdesc="Download the AUR packages you trust and use pacman to install them."

--- a/local-aur
+++ b/local-aur
@@ -68,7 +68,7 @@ def build_package(pkgname: str) -> None:
             for file in files:
                 dest = os.path.join(DATA, 'repo', os.path.basename(file))
                 dest_files.append(dest)
-                os.rename(quote(file), quote(dest))
+                shutil.move(quote(file), quote(dest))
 
             json_data['packages'][pkgname] = {
                 'commit_hash': get_commit_hash(pkgname),

